packages/elements/list/src/getListDeleteForward.ts

```diff
@@ -70,11 +143,11 @@ export const getListDeleteForward = (editor: SPEditor) => {
         listItem: siblingListItem,
         reverse: false,
       });
-      if (moved) return;
```

Comment:
This was intentional so that if more transformations are added it's not forgotten. Not that it's super important, but it's a trailing comma kind of thing.

Reply:
It should either be `if (moved) return moved;` or the next executable line is `return;` that makes this unnecessary..
But based on the passing tests it shouldn't be `if (moved) return moved;`

Reply:
To clarify - if moved, the function should terminate there. If we were to omit it, there is no explicit intent to terminate the function, but rather it's coincidental that it does terminate because it is the last call. If another call was added after, then it's up to the author to add `if (moved) return;` based on previous code. Whether they'll do that is an unknown.

But since i wrote the explicit return, i'm saying that the function should terminate if something happened as a result of the previous call. It's explicit on purpose, but since it's adding confusion i see that i should have done it differently. Problem is that i was following the style of the two delete functions, so a refactor would need to be done to both. Since i didn't wan't to expand the scope, i just did this, and i think it should be there :)

Reply:
Sorry, my English was really of yesterday..
[These are the lines in question](https://github.com/udecode/plate/blob/603601195c92ee239699df875b377ff64de81673/packages/elements/list/src/getListDeleteForward.ts#L73-L75), I removed L73 since the next executable line is L75 which is a `return;`

I get that there could be some code between L73 and L75 if `moved` is false, but since there is none I feel that line is really not necessary, I think in this case it will be the job of the future contributor to early return if needed.

You have something similar in L88 where you early return, but there you have something after it.

Reply:
I believe both code blocks should have `if (moved) return;` - the fact that it's missing after L90 is a mistake. What a future contributor does is out of our hands, but in both cases mentioned the function should exit if moved is true. The return might be redundant but it shows an explicit intent to exit if true, and an explicit exit makes it easier to deal with in the future.

I have no alternative off the top of my head to make the intent explicit in code without refactoring everything using those methods. The only two ways i see to tell the future contributor to exit if true before adding logic are:

1. A comment
2. Code that the future contributor would actually write (`if (moved) return;`) having to delete the comment first

I'm not married to the return, but i don't think leaving something to be decided by someone in the future is the solution. I'm open to a compromise though.